services/QuillLMS/client/app/bundles/Shared/components/studentLessons/register.tsx

```diff
@@ -75,13 +76,43 @@ class Register extends React.Component<any, any> {
     );
   }
 
+  translatedText = (): string => {
+    function getTranslation( translations, targetLanguage): string | undefined {
```

Comment:
I would move this inner function outside of `translatedText` (and maybe even outside of the component?) for readability and theoretical performance concerns about re-defining `getTranslation` every time `translatedText` gets called.